release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -0,0 +1,1534 @@
+access to ES for those tools that want to access its data.
+
+You now have direct access to ES using only your OpenShift token and have the
+ability to provide the external ES and ES Ops hostnames when creating the server
+certificate (similar to Kibana). Ansible tasks now simplify route deployment.
+
+[[ocp-37-removed-metrics-and-logging-deployers]]
+==== Removed Metrics and Logging Deployers
+
+The metrics and logging deployers bare now replaced with `playbook2image` for
+`oc cluster up` so that `openshift-ansible` is used to install logging and
+metrics:
+
+----
+$ oc cluster up --logging --metrics
+----
+
+Check metrics and pod status:
+
+----
+$ oc get pod -n openshift-infra
+$ oc get pod -n logging
+----
+
+[[ocp-37-prometheus]]
+==== Prometheus (Technology Preview)
+
+{product-title} operators deploy Prometheus (currently in
+xref:ocp-37-technology-preview[Technology Preview] on a {product-title} cluster,
+collect Kubernetes and infrastructure metrics, and get alerts. Operators can see
+and query metrics and alerts on the Prometheus web dashboard, or bring their own
+Grafana and hook it up to Prometheus.
+
+See xref:../nstall_config/cluster_metrics.adoc#openshift-prometheus[Prometheus
+on OpenShift] for more information.
+
+[[ocp-37-integrated-approach-to-adding-hosa]]
+==== Integrated Approach to Adding Hawkular OpenShift Agent (Techology Preview)
+
+Hawkular OpenShift Agent (HOSA) remains in
+xref:ocp-37-technology-preview[Technology Preview]. It is packaged and can now
+be installed with the `openshift_metrics_install_hawkular_agent` option in the
+installer by setting it to `true`.
+
+See
+xref:../install_config/cluster_metrics.adoc#metrics-ansible-variable[Enabling
+Cluster Metrics] for more information.
+
+[[ocp-37-developer-experience]]
+=== Developer Experience
+
+[[ocp-37-template-instantation-api]]
+==== Template Instantiation API
+
+Clients can now easily invoke a server API instead of relying on client logic.
+
+* To set parameters, create a secret with values.
+* Create a `TemplateInstance` containing the whole template you want to
+instantiate, and a reference to the secret created above.
+* Poll the `TemplateInstance` or watch the `TemplateInstance` until either the
+`Ready` or `InstantiateFailure` condition types report status `True`.
+
+----
+$ curl -k \
+    -X POST \
+    -d @-  \
+    -H "Authorization: Bearer $Token" \
+    -H 'Accept: application/json' \
+    -H 'Content-Type: application/json' \
+    https://$ENDPOINT/apis/template.openshift.io/v1/templateinstances <<'EOF'
+{
+  "kind": "TemplateInstance",
+  "apiVersion": "template.openshift.io/v1",
+  ...
+  EOF
+----
+
+[[ocp-37-dev-experience-metrics]]
+==== Metrics
+
+{product-title} now includes needed Prometheus monitoring and alerting. Expose
+build step timings (time to pull images, fetch sources, run assemble, commit
+images, push images). Expose failure reasons (for example, see that builds are
+consistently failing due to failure to fetch source).
```

Comment:
the build step timings are not exposed as prometheus metrics.  So this is two things:

1) prometheus metrics that show you the health of builds in the system (number running, failing, failure reasons, etc)
2) timing information on build objects themselves to show how long they spent in various steps (*not* exposed as prometheus metrics)